### DISCUSSION TITLE
2EA6D9A9-1DA9-AA6F-91DC-C384D60813D0

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#124] [General]
# 2EA6D9A9-1DA9-AA6F-91DC-C384D60813D0

The addon adds a lot of massive bright stars, nebulae and clusters in the Milky Way that are obscured by a lot of dust in the way.

https://celestia.mobi/addon?item=2EA6D9A9-1DA9-AA6F-91DC-C384D60813D0

**Comment by MarsEmpire123** [comment]
wow its a nice add-on
